cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "Banner",
+    "description": "A banner.",
+    "group": "commerce_assets",
+    "attribute_definition_groups": [
+      {
+       "id":"image",
+       "name":"Banner Image Configuration",
```

Comment:
Watch out for the indentations and formatting, even in json files

Reply:
Fixed